donut-app/source/GoogleStoreFinder.mc

```diff
@@ -0,0 +1,207 @@
+using Toybox.Communications as Comm;
+using Toybox.WatchUi as Ui;
+
+// A class used to get the nearest dunkin donuts using the Google API
+class GoogleStoreFinder {
+	// A good response from a server is 200
+	const OK_RESPONSE_CODE = 200;
+	// The google API key to request the nearby and distance matrix
+	//  Unless you're me (the developer), you don't get to know my key.
+	//  I'm trying to keep it off github, help me out will ya?
+	const API_KEY = "";
+	
+	// Use this variable to rate limit my google maps API calls...
+    hidden var webRequests = 0;
+    
+    // The Google place_id of the nearest dunkin donuts
+	hidden var nearestPlaceId = null;
+	// A array of 2 elements = [latitude, longitude]
+	hidden var currentPosition = null;
+	
+	// A function call used to set the text to the nearest dunkin
```

Comment:
Technically, it's a method NOT a function call. Update these comments respectively.